index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
+            </p>
+
+            <h3>Relation to the VC Data Model</h3>
+            <p>
+                This specification defines encoding rules of the verifiable credential and presentation data
+                model onto JWT and JWS. It further defines processing rules how and when to make use of specific
+                JWT registered claim names and specific JWS registered header parameter names to allow systems based
+                on JWT to comply with this specification. If these specific claim names and header parameters are
+                present, their respective counterpart in the verifiable credential and presentation can be omitted
+                to avoid duplication.
+            </p>
+
+            <h3>IANA Considerations</h3>
+            <p>
+                This specification introduces two new registered claim names which contain the JSON or JSON-LD
+                verifiable credentials and presentation object enclosed in the JWT payload:
+                <ul>
+                    <li><code>vc</code>: JSON object, and MUST be present in a JWT verifiable credential.
+                    <li><code>vcp</code>: JSON object, and MUST be present in a JWT verifiable presentation.
+                        To allow JWT verifiable credentials to be included in the verifiable presentation,
+                        a new property <code>vc_jwt</code> is introduced that can contain an array of
+                        JWT verifiable credentials.
+                    </li>
+                </ul>
+            </p>
+
+            <h3>JWT and JWS Considerations</h3>
+            <p>
+                If the JWS is present, the digital signature either refers to the issuer of the verifiable credential,
+                or the holder of the credential in case of a presentation. The JWS will prove that the contained
+                JWT payload was signed by the issuer of the JWT and therefore, the <code>proof</code> property can be
+                omitted. If no JWS is present, then a proof attribute MUST be provided. The <code>proof</code> property
+                can be used to represent more complex proofs e.g., if the creator is different from the issuer, or a
+                <code>proof</code> which is not based on digital signatures such as proof of work. The issuer MAY
+                include both, a JWS and a <code>proof</code> property. For backwards compatibility reasons, the issuer
+                MUST use JWS to represent proofs based on a digital signature.
+            </p>
+            <p>
+                In addition, the following JOSE headers MUST be present in the JWS:
+                <ul>
+                    <li><code>alg</code> MUST be used for RSA and ECDSA based digital signatures.</li>
+                    <li><code>kid</code> MUST be used if there are multiple keys associated with the issuer of the JWT.
```

Comment:
I don't think this is needed. It's an option but adds extra verbosity. Our implementation checks the DID document and tests each public key. This works fine in almost all use cases and removes duplication.

Reply:
- Agree with MUST. 
- I also would prefer to see this in a separate document. 
- `kid` MAY be used.

Reply:
👍